bundle/regal/ast/search.rego

```diff
@@ -126,11 +119,9 @@ _find_vars(value, last) := {"assign": _find_assign_vars(value)} if {
 	last == "terms"
 	value[0].type == "ref"
 	value[0].value[0].type == "var"
-	value[0].value[0].value == "eq"
+	value[0].value[0].value in {"assign", "eq"}
```

Comment:
That's a function call, too, isn't it? It'll become `internal.member_1(value[0].value[0].value, {"assign", "eq"})` or some such thing IIRC